CHANGELOG for v1.x.x.md

```diff
@@ -2,6 +2,382 @@
 
 #### This changelog consists the bug & security fixes and new features being included in the releases listed below.
 
+## **v1.1.3 (19th of Jine 2020)** - *Release*
```

Comment:
This should be June instead of Jine @jitendra-webkul

Reply:
Done in this PR #3282